Docker/Dockerfile.content-publishing

```diff
@@ -7,9 +7,9 @@ COPY package* tsconfig* nest-cli.json ./
 
 RUN npm ci
 
-WORKDIR /build/libs/content-publishing-lib
+WORKDIR /build/libs
```

Comment:
No, because the final image doesn't keep it. The build stage is a separate image that just facilitates building the final (smaller) image. (See line 28 where the production image comes from a clean `node:20` base)